test/verify/machineslib.py

```diff
@@ -1258,6 +1258,19 @@ def installWithErrorTest(dialog):
                                              os_name=config.NOVELL_NETWARE_6,
                                              start_vm=False))
 
+            self.machine.execute("qemu-img create -f qcow2 /var/lib/libvirt/vmTest20.qcow2 128M")
+            self.machine.execute("virsh pool-refresh libvirt")
+
+            createTest(TestMachines.VmDialog(self, "subVmTestCreate20", sourceType='pxe',
+                                             location="Host Device {0}: macvtap".format(iface),
+                                             memory_size=50, memory_size_unit='MiB',
+                                             storage_size=0, storage_size_unit='MiB',
+                                             os_vendor=config.NOVELL_VENDOR,
+                                             os_name=config.NOVELL_NETWARE_6,
+                                             storage_pool="libvirt",
+                                             storage_volume="vmTest20.qcow2",
+                                             start_vm=False,))
```

Comment:
I would rather have here start_vm=True. It performs a completely logic test. We already test starting VMs from local images in subVmTestCreate17.
Also if you can add a test for separate install step that would be awesome!